src/BuildIt.ML/Samples/BuildIt.ML.Sample.Android/MainActivity.cs

```diff
@@ -2,21 +2,26 @@
 using Android.Content.PM;
 using Android.OS;
 using BuildIt.ML.Sample.UI;
+using Plugin.CurrentActivity;
+using Plugin.Permissions;
 
 namespace BuildIt.ML.Sample.Droid
 {
     [Activity(Label = "BuildIt.ML.Sample", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
-    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+    public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsApplicationActivity
     {
         protected override void OnCreate(Bundle bundle)
         {
-            TabLayoutResource = Resource.Layout.Tabbar;
-            ToolbarResource = Resource.Layout.Toolbar;
-
             base.OnCreate(bundle);
 
             global::Xamarin.Forms.Forms.Init(this, bundle);
+            CrossCurrentActivity.Current.Init(this, bundle);
```

Comment:
Is there no way that we can get around having this call here?

Reply:
to be clear, the camera plugin depends on the permissions plugin, which in turns depends on that activity one. so no, it won't go away in the sample if we keep using the permissions plugin (which we do for BuildIt.Forms)